src/Http/Controllers/V1/Admin/Velocity/ContentController.php

```diff
@@ -109,4 +110,24 @@ public function massUpdate(MassUpdateRequest $request)
             'message' => __('rest-api::app.common-response.success.mass-operations.update', ['name' => 'content']),
         ]);
     }
+
+    /**
+     * To mass remove the contents.
+     *
+     * @param  \Webkul\Core\Http\Requests\MassDestroyRequest  $request
+     * @return \Illuminate\Http\Response
+     */
+
+    public function massDestroyResources(MassDestroyRequest $request)
+    {
+        foreach ($request->indexes as $id) {
+            $this->getRepositoryInstance()->findOrFail($id);
+
+            $this->getRepositoryInstance()->delete($id);
+        }
+
+        return response([
+            'message' => __('rest-api::app.common-response.success.delete', ['name' => 'Content']),
+        ]);
+    }
```

Comment:
Refer this pull request review,
https://github.com/bagisto/rest-api/pull/42#pullrequestreview-1377822983